chatkit-android/src/main/kotlin/com/pusher/chatkit/AndroidChatkitDependencies.kt

```diff
@@ -1,33 +1,30 @@
 package com.pusher.chatkit
 
 import android.content.Context
-import com.pusher.SdkInfo
-import com.pusher.platform.AndroidDependencies
-import com.pusher.platform.PlatformDependencies
+import com.pusher.platform.AndroidMediaTypeResolver
+import com.pusher.platform.MediaTypeResolver
 import com.pusher.platform.logger.Logger
 import com.pusher.platform.tokenProvider.TokenProvider
-import okhttp3.OkHttpClient
 
 /**
- * [ChatkitDependencies] implementation for Android using [AndroidDependencies] to fulfil [PlatformDependencies].
+ * Dependencies used to configure the Chatkit SDK, primarily to set your Android [Context] and
+ * [TokenProvider], as well as optionally other dependencies like your [Logger].
+ *
+ * @param context Android context used for string formatting in ViewModels respecting
+ * device settings and for push notifications (usually [android.app.Application] but if you use
+ * the SDK just in a shorter-lived scope like [android.app.Activity] you can pass it as well)
+ * @param tokenProvider for providing your JWT auth tokens, depending on your backend you may
+ * consider using the exiting [ChatkitTokenProvider] implementation
+ * @param logger pass if you want the SDK to log messages
+ * @param mediaTypeResolver pass if you upload files for message attachments so that
+ * HTTP Content-Type is set
```

Comment:
I suspect that the customer will pretty much never want to provide this, it only exists here so that the Android specific functionality can be wrapped in a pure-JVM interface for use in the core module.

Reply:
good one! indeed it is very unlikely scenario someone would like to provide their custom one so let's not complicate the API even if it's just an optional. if any user requests that then we can consider adding it. done: c7a0db71a666bc39d6de666ad42b2483b61e3f28